rpc/client/client.go

```diff
@@ -750,13 +747,14 @@ func (c *Client) Status(ctx context.Context) (*ctypes.ResultStatus, error) {
 			LatestAppHash:     latestAppHash[:],
 			LatestBlockHeight: int64(latestHeight),
 			LatestBlockTime:   time.Unix(0, int64(latestBlockTimeNano)),
+			// CatchingUp is true if the node is not at the latest height received from p2p or da.
```

Comment:
shuld be on the field on the struct, not where the value is set!

Reply:
should I fork Tendermint then?

Reply:
ah sorry didn't realise it was tendermint
all good